app/models/form/lettings/questions/rsnvac.rb

```diff
@@ -7,47 +7,98 @@ def initialize(id, hsh, page)
     @type = "radio"
     @check_answers_card_number = 0
     @hint_text = ""
-    @answer_options = ANSWER_OPTIONS
     @question_number = 17
   end
 
-  ANSWER_OPTIONS = {
-    "13" => {
-      "value" => "Internal transfer",
-      "hint" => "Excluding renewals of a fixed-term tenancy",
-    },
-    "5" => {
-      "value" => "Previous tenant died with no succession",
-    },
-    "9" => {
-      "value" => "Re-let to tenant who occupied same property as temporary accommodation",
-    },
-    "14" => {
-      "value" => "Renewal of fixed-term tenancy",
-    },
-    "19" => {
-      "value" => "Tenant involved in a succession downsize",
-    },
-    "8" => {
-      "value" => "Tenant moved to private sector or other accommodation",
-    },
-    "12" => {
-      "value" => "Tenant moved to other social housing provider",
-    },
-    "18" => {
-      "value" => "Tenant moved to care home",
-    },
-    "20" => {
-      "value" => "Tenant moved to long-stay hospital or similar institution",
-    },
-    "6" => {
-      "value" => "Tenant abandoned property",
-    },
-    "10" => {
-      "value" => "Tenant was evicted due to rent arrears",
-    },
-    "11" => {
-      "value" => "Tenant was evicted due to anti-social behaviour",
-    },
-  }.freeze
+  def answer_options
+    if form.start_year_after_2024?
```

Comment:
I initially did this more DRYly, but it ended up being way less readable, mainly because the answer ordering has changed